Portfolio/src/App.vue

```diff
@@ -101,23 +100,39 @@ body {
 .navbar {
     display: flex;
     justify-content: flex-end;
-    margin: 4rem 5rem;
-    font-size: 43px;
+    font-size: 25px;
     gap: 60px;
     list-style: none;
 }
 
+.openedMenu {
+    display: none;
+}
+
+
 @media (min-width: 280px) and (max-width: 767px) {
     .navbar {
         display: none;
+
     }
 
-    .openedMenu {
+    .line {
+        width: 55px;
+        height: 3px;
+        background-color: black;
+        margin: 3px;
+    }
+
+    .whiteLine {
```

Comment:
It's generally a bad idea to name it like that `whiteLline`, since it hard-couples your styles with the html. 
Consider that you may want to change the color of the line to grey, then you would have to change both the style, the className **and** your html.
On the contrary, if you called it `colored-line` or `spacer` (depending on what it is) you can simply change the background-color without touching the name.

Reply:
Done